templates/firefox-ios.xliff

```diff
@@ -3707,6 +3731,279 @@ Private Tab</source>
+
+Using the power of %5$@ by %6$@, we help you avoid biased and inauthentic reviews. Our AI model is always improving to protect you as you shop.</source>
+        <note>Label for the first paragraph of the Shopping Experience Opt In onboarding Card (Fakespot). The first parameter will be the website the user is coming from when viewing this screen (default Amazon). The second parameter will be replaced by the app name. The third and fourth parameters will be the other two websites that are currently supported (Amazon, Best Buy or Walmart) besides the one used for the first parameter. The fifth and sixth parameter are the Fakespot app name and the company name of Mozilla.</note>
+      </trans-unit>
+      <trans-unit id="Shopping.OptInCard.HeaderLabel.Title.v120" xml:space="preserve">
+        <source>Try our trusted guide to product reviews</source>
+        <note>Label for the header of the Shopping Experience Opt In onboarding Card (Fakespot)</note>
+      </trans-unit>
+      <trans-unit id="Shopping.OptInCard.LearnMoreButtonTitle.Title.v120" xml:space="preserve">
+        <source>Learn more</source>
+        <note>Label for the Learn more button in the Shopping Experience Opt In onboarding Card (Fakespot)</note>
+      </trans-unit>
+      <trans-unit id="Shopping.OptInCard.MainButton.Title.v120" xml:space="preserve">
+        <source>Yes, Try It</source>
+        <note>Text for the main button of the Shopping Experience Opt In onboarding Card (Fakespot)</note>
+      </trans-unit>
+      <trans-unit id="Shopping.OptInCard.PrivacyPolicy.Button.Title.v120" xml:space="preserve">
+        <source>Privacy policy</source>
+        <note>Show Firefox Browser Privacy Policy page from the Privacy section in the Shopping Experience Opt In onboarding Card (Fakespot). See https://www.mozilla.org/privacy/firefox/</note>
+      </trans-unit>
+      <trans-unit id="Shopping.OptInCard.SecondaryButton.Title.v120" xml:space="preserve">
+        <source>Not now</source>
+        <note>Text for the secondary button of the Shopping Experience Opt In onboarding Card (Fakespot)</note>
+      </trans-unit>
+      <trans-unit id="Shopping.OptInCard.TermsOfUse.Button.Title.v120" xml:space="preserve">
+        <source>Terms of use</source>
+        <note>Show Firefox Browser Terms of Use page from the Privacy section in the Shopping Experience Opt In onboarding Card (Fakespot). See https://www.mozilla.org/privacy/firefox/</note>
+      </trans-unit>
+      <trans-unit id="Shopping.ReliabilityScore.Grade.A11y.Label.v120" xml:space="preserve">
+        <source>Grade %@</source>
+        <note>Accessibility label for the Grade labels used in 'How we determine review quality' card and 'How reliable are these reviews' card displayed in the shopping review quality bottom sheet. The placeholder will be replaced by a grade letter (e.g. A).</note>
```

Comment:
I'd just add that it's a grading system from A-F

Reply:
Will be fixed in https://github.com/mozilla-mobile/firefox-ios/pull/16964